test/pummel/test-crypto-timing-safe-equal-benchmarks.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
```

Comment:
Nit: would you mind to move this after the `common.skip()` conditions?

Reply:
I'd love to. 👍